src/main/java/net/bhardy/bizzo/billing/impl/PolicyBuilderImpl.java

```diff
@@ -36,6 +36,9 @@
  * @see net.bhardy.bizzo.billing.PolicyBuilder
  */
 public class PolicyBuilderImpl implements PolicyBuilder {
+
+    public int[] PROBLEM = null;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make PROBLEM a static final constant or non-public and provide accessors if needed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "PROBLEM" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00116)